airbyte-integrations/connectors/source-s3/acceptance-test-config.yml

```diff
@@ -6,18 +6,21 @@ tests:
     - spec_path: "integration_tests/spec.json"
   connection:
     # for CSV format
-    - config_path: "secrets/config.json"
+    - config_path: "integration_tests/config_minio.json"
       status: "succeed"
     # for Parquet format
     - config_path: "secrets/parquet_config.json"
       status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
```

Comment:
Please leave at least one check with the result "failed"

Reply:
oops accidentally removed it